web-client/scripts/build.sh

```diff
@@ -1,17 +1,39 @@
+#!/bin/bash
+
 set -e
 
 # TODO: Disable rust compiler warnings with [lints] table in Cargo.toml when it becomes stable in 1.75 (https://github.com/rust-lang/cargo/issues/12115)
 
+# Defaults
 CARGO_PROFILE="release-wasm"
 CARGO_TARGET="wasm32-unknown-unknown"
 CARGO_OUTPUT="../target/$CARGO_TARGET/$CARGO_PROFILE/nimiq_web_client.wasm"
```

Comment:
This would need to be set again if `$CARGO_TARGET` or `$CARGO_PROFILE` changes.

Reply:
Oh wait, I did not read my own code :D They change with command line arguments. Hold on...

Reply:
I moved the definition of `CARGO_OUTPUT` to after the arguments-handling, setting the default if it was not set from an argument. Is that right?

Reply:
Looks correct, thanks!